src/sdk/pynni/nni/batch_tuner/batch_tuner.py

```diff
@@ -88,13 +85,40 @@ def generate_parameters(self, parameter_id):
         ----------
         parameter_id : int
         """
-        self.count +=1
-        if self.count>len(self.values)-1:
+        self.count += 1
+        if self.count > len(self.values) - 1:
             raise nni.NoMoreTrialError('no more parameters now.')
         return self.values[self.count]
 
     def receive_trial_result(self, parameter_id, parameters, value):
         pass
 
     def import_data(self, data):
-        pass
+        """Import additional data for tuning
+        Parameters
+        ----------
+        data:
+            a list of dictionarys, each of which has at least two keys, 'parameter' and 'value'
+        """
+        if len(self.values) == 0:
+            logger.info("Search space has not been initialized, skip this data import")
+            return
+
+        self.values = self.values[(self.count+1):]
+        self.count = -1
+
+        _completed_num = 0
```

Comment:
In NNI code, normally there is no underscore "_" at the begining of local variable names,  suggest to remove it for consistency.